packages/widgets/src/components/TradingScoreboard/types.ts

```diff
@@ -0,0 +1,18 @@
+import { ITradingInfoCardProps } from '@components/TradingInfoCard/types';
+import { SerializedStyles } from '@emotion/react';
+
+type TradingScoreboardType = Pick<
```

Comment:
Let's use this name: `TradingScoreboardItem`

Reply:
Done